### PR TITLE
fix: Add the position of the icon in multiple mode of select component

### LINKED ISSUE #20887: bug: select 组件 多选模式下选项占用 清空按钮位置
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/lively-dust-pn1n9)

### Steps to reproduce
- 鼠标移到框内
- 观察全清的按钮

### What is expected?
- 按钮位置不遮挡

### What is actually happening?
- 按钮位置遮挡

| Environment | Info |
|---|---|
| antd | 4.0.0-rc.1 |
| React | The latest |
| System | mac |
| Browser | chrome |

---
- `ant-select-selector` 的 padding 只有 4px，不管是 默认 `large` `small` 都会出现 选项遮挡 清空 按钮的情况
- 考虑 是否 可将 padding 增加

### PR DESCRIPTION
**Pull request by xrkffgg**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close https://github.com/ant-design/ant-design/issues/20887

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Add the position of the icon in multiple mode of select component; Added radio RTL mode, free position         |
| 🇨🇳 Chinese |增加 Select 组件 多选模式下，清空按钮的空间位置；增加单选RTL 模式下，空余位置           |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/select/style/single.less [hunk 1]
```diff
@@ -160,6 +160,7 @@
 
         .@{select-prefix-cls}-rtl& {
           right: 0;
+          left: @input-padding-horizontal-sm + @font-size-base * 1.5;
```
Comment (afc163) [inline components/select/style/single.less]: 1.5 是 line-height ？

Comment (xrkffgg) [reply components/select/style/single.less]: 我是根据 对应的 来写的，你看下 159 行的 定义

Comment (afc163) [reply components/select/style/single.less]: 如果这个值是表示 line-height，那么可以直接用 @line-height-base 变量。

Comment (afc163) [reply components/select/style/single.less]: 如果不是就不用改。

Comment (xrkffgg) [reply components/select/style/single.less]: 那 原有代码中 的 1.5 我也改成 @line-height-base ？

**Comment by abenhamdine** [comment]
same as https://github.com/ant-design/ant-design/pull/20914 ?

File: components/select/style/single.less (hunk 1 above)
Comment (zombieJ) [reply components/select/style/single.less]: 这里的 `1.5` 不是 `line-height`，是根据 `font-size` 算一个宽度。要不然提到上面去搞成一个变量，免得误导了？

**Comment by zombieJ** [comment]
> same as #20914 ?

Enhancement with it.

**Pull request closed without merging**